pkg/apis/apps/v1beta2/conversion_test.go

```diff
@@ -70,13 +70,19 @@ func TestV1beta2StatefulSetSpecConversion(t *testing.T) {
 				Template:            apiTemplate,
 				ServiceName:         "foo",
 				PodManagementPolicy: apps.PodManagementPolicyType("bar"),
+				PersistentVolumeClaimPolicy: &apps.StatefulSetPersistentVolumeClaimPolicy{
```

Comment:
I suppose we should only test the custom conversions.... interestingly, I don't see that being tested :/ (not related to your change).

Reply:
done (ie, removed the test)